model/config.go

```diff
@@ -2522,6 +2543,20 @@ func (ds *DisplaySettings) isValid() *AppError {
 	return nil
 }
 
+func (ss *SupportSettings) isValid() *AppError {
+	if *ss.CustomServiceTermsEnabled && len(*ss.CustomServiceTermsText) == 0 {
```

Comment:
This seems troublesome, given that we enforce clearing out the service terms text in various config methods up above. I'll summarize this in a high level comment.

Reply:
Didn't get that.